docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -0,0 +1,127 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ledger-api-test-tool:
+
+Ledger API Test Tool
+####################
+
+The Ledger API Test Tool is a command line tool for testing the correctness of
+ledger implementations based on DAML and the :doc:`Ledger API
+</app-dev/ledger-api-introduction/index>`. It’s useful for ledger implementation
+developers, who are using the DAML Ledger Implementation Kit to develop a DAML
+Ledger on top of their distributed-ledger or database of choice.
+
+Use this tool to verify if the Ledger API endpoint conforms to the :doc:`DA
+Ledger Model </concepts/ledger-model/index>`.
+
+.. contents:: On this page:
+  :local:
+
+Downloading the tool
+====================
+
+TODO
+
+Extracting :code:`.dar` file required to run the tests
+======================================================
+
+Before you can run the Ledger API test tool on your ledger, you need to load a
+specific set of DAML templates onto your ledger.
+
+#. Obtain the corresponding :code:`.dar` file with:
+
+   ``ledger-api-test-tool -x``
```

Comment:
I'd add a line after this that says "This creates a file ``SemanticTests.dar``" Then, in the next point, you can just say "Load ``SemanticTests.dar`` into your ledger"